src/storage/heap_file.c

```diff
@@ -12792,6 +12792,8 @@ heap_attrvalue_get_index (int value_index, ATTR_ID * attrid, int *n_btids, BTID
     {
       value = &idx_attrinfo->values[value_index];
       *n_btids = value->last_attrepr->n_btids;
+      // TODO : OR_ATTRIBUTE structure has been changed due to CUBVEC-142 issue.
```

Comment:
```suggestion
      // TODO (CUBVEC) : OR_ATTRIBUTE structure has been changed due to CUBVEC-142 issue.
```

나중에 grep으로 편하게 추적하기 위해서, 동일한 포맷으로 작성